framework-integrations/sagemaker/training/code/conf/config.yaml

```diff
@@ -0,0 +1,19 @@
+# The Defaults List is a list in an input config that instructs Hydra how to build the output config.
+defaults:
+  - _self_
+  - dataset: uniref50
```

Comment:
`uniref100+.yaml` should be the default.

Reply:
For sagemaker training we had used the uniref50 as sample dataset. You can change this to any training dataset.